src/web/pages/hosts/dashboard/HostsVulnScoreDisplay.jsx

```diff
@@ -15,20 +17,22 @@ import DataTableDisplay from 'web/components/dashboard/display/DataTableDisplay'
 import {riskFactorColorScale} from 'web/components/dashboard/display/utils';
 import withFilterSelection from 'web/components/dashboard/display/withFilterSelection';
 import {registerDisplay} from 'web/components/dashboard/Registry';
+import useGmp from 'web/hooks/useGmp';
 import {HostsVulnScoreLoader} from 'web/pages/hosts/dashboard/Loaders';
-import compose from 'web/utils/Compose';
 import PropTypes from 'web/utils/PropTypes';
 import {resultSeverityRiskFactor} from 'web/utils/severity';
 import {formattedUserSettingLongDate} from 'web/utils/userSettingTimeDateFormatters';
-import {withRouter} from 'web/utils/withRouter';
 
 const ToolTip = styled.div`
   font-weight: normal;
   text-align: center;
   line-height: 1.2em;
 `;
 
-const transformVulnScoreData = (data = {}) => {
+const transformVulnScoreData = (
+  data = {},
+  {severityRating = DEFAULT_SEVERITY_RATING},
```

Comment:
Default parameter should be after.
```suggestion
  {severityRating = DEFAULT_SEVERITY_RATING},
  data = {},

```

Reply:
That doesn't work here. The function is generic. First argument is the data from the backend and second arg contains props passed trough DataDisplay into the function. I am not sure anymore why data is set to an empty object by default but i guess it's to avoid issues when the data is loaded currently.